x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/kpi/index.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-export * from './authentications';
+export * from '../../users/kpi/authentications';
```

Comment:
Looks weird to export the `users` authentications from a `hosts` kip factory, is it really needed?

Reply:
Good catch! I forgot to delete this line when I moved the file to the users folder.